.github/ci_commit_pins/xla.txt

```diff
@@ -1 +1 @@
-709c75a04d9b35d83a9509e1534a8aa2046b8912
+c3c16ccac41cb2db6ba88fb31342f4af62c7e15a
```

Comment:
Should not be there?

Reply:
This is for https://github.com/pytorch/xla/pull/5434, xla was relying on indirectly including iostream through our headers.